static/src/javascripts/bootstraps/facia.js

```diff
@@ -114,17 +118,19 @@ define([
         ready = function () {
             if (!this.initialised) {
                 this.initialised = true;
-                accessibility.shouldHideFlashingElements();
-                modules.showSnaps();
-                modules.showContainerShowMore();
-                modules.showContainerToggle();
-                modules.upgradeMostPopularToGeo();
-                lazyLoadContainers();
-                stocks();
-                sponsorship();
-                modules.showWeather();
-                modules.showLiveblogUpdates();
-                modules.startSlideshow();
+                background(robust.makeBlocks([
+                    ['f-accessibility', accessibility.shouldHideFlashingElements],
+                    ['f-snaps', modules.showSnaps],
+                    ['f-show-more', modules.showContainerShowMore],
+                    ['f-container-toggle', modules.showContainerToggle],
+                    ['f-geo-most-popular', modules.upgradeMostPopularToGeo],
+                    ['f-lazy-load-containers', lazyLoadContainers],
+                    ['f-stocks', stocks],
+                    ['f-sponsorship', sponsorship],
+                    ['f-weather', modules.showWeather],
+                    ['f-live-blog-updates', modules.showLiveblogUpdates],
+                    ['f-slideshow', modules.startSlideshow]
+                ]));
```

Comment:
good point thanks!  although I do still want to add the next animation frame even if there's an error.  Maybe if I do that first it would cover that possibility though - I'll do that.